lib/author.rb

```diff
@@ -0,0 +1,22 @@
+class Author
+  attr_reader :first_name, :last_name, :books
+
+  def initialize(info)
+    @first_name = info[:first_name]
+    @last_name = info[:last_name]
+    @books = []
+  end
+
+  def name
+    "#{@first_name} #{@last_name}"
+  end
+
+  def add_book(book)
+    @books.push(book)
+  end
```

Comment:
You should not be creating a book method, write should be also adding the book to the instance variable.